debian/tests/control

```diff
@@ -1,4 +1,9 @@
 # Skip integration tests when building package: they need docker images.
-Test-Command: ADSYS_SKIP_INTEGRATION_TESTS=1 go test -v -mod=vendor ./...
+Test-Command: ADSYS_SKIP_INTEGRATION_TESTS=1 ADSYS_SKIP_SUDO_TESTS=1 go test -v -mod=vendor ./...
 Restrictions: allow-stderr
 Depends: @builddeps@
+
+# Run tests that require root.
+Test-Command: go test -v -mod=vendor ./cmd/adwatchd/integration_tests ./internal/watchdtui
```

Comment:
I wonder if we can use the sudo_packages list exported somewhere so that we source it here and in CI (to avoid repetition and forgetting to update one place if the list grows). Not a blocker though.